`vscode@office:/nav2_ros_gz$ colcon build --symlink-install --packages-skip nav2_system_tests
Starting >>> nav2_common
Starting >>> nav_2d_msgs
Starting >>> gz_ros2_control
Starting >>> turtlebot3_gazebo
Starting >>> turtlebot3_fake_node
Finished <<< turtlebot3_gazebo [0.63s]                                                                                                                                    
Finished <<< turtlebot3_fake_node [0.62s]                                                                                                                          
Starting >>> turtlebot3_simulations
Finished <<< gz_ros2_control [0.74s]
Starting >>> gz_ros2_control_demos
Starting >>> gz_ros2_control_tests
Starting >>> turtlebot3_gz                                                                                                                                            
Finished <<< nav2_common [1.04s]                                                                                                           
Starting >>> nav2_msgs
Starting >>> nav2_voxel_grid
Finished <<< turtlebot3_simulations [0.43s]                                                                                                             
Finished <<< gz_ros2_control_tests [0.62s]                                                                                                             
Finished <<< turtlebot3_gz [0.69s]                                                                                                                                       
Finished <<< gz_ros2_control_demos [0.85s]                                                                                                                                
Finished <<< nav2_voxel_grid [0.71s]                                                                                             
Finished <<< nav_2d_msgs [2.08s]                                                             
Starting >>> dwb_msgs
Finished <<< dwb_msgs [1.38s]                                                                 
Finished <<< nav2_msgs [3.18s]                    
Starting >>> nav2_util
Starting >>> nav2_simple_commander        
Finished <<< nav2_util [0.51s]                                                                  
Starting >>> nav2_behavior_tree
Starting >>> nav_2d_utils
Starting >>> nav2_lifecycle_manager
Starting >>> nav2_map_server
Starting >>> nav2_amcl
Starting >>> nav2_velocity_smoother
Finished <<< nav2_velocity_smoother [1.00s]                                                                                                                               
Finished <<< nav2_lifecycle_manager [1.19s]                                                                                               
Finished <<< nav_2d_utils [1.23s]
Starting >>> nav2_rviz_plugins
Finished <<< nav2_amcl [1.26s]                                                                                                                                          
--- stderr: nav2_simple_commander
/usr/lib/python3/dist-packages/setuptools/command/easy_install.py:158: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
/usr/lib/python3/dist-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
---
Finished <<< nav2_simple_commander [1.88s]
Finished <<< nav2_map_server [1.46s]                                                                                                           
Starting >>> nav2_costmap_2d
Finished <<< nav2_rviz_plugins [0.88s]                                                                                                         
Finished <<< nav2_behavior_tree [3.52s]                                                                      
--- stderr: nav2_costmap_2d                                 
In file included from /nav2_ros_gz/src/navigation2/nav2_costmap_2d/test/integration/test_costmap_subscriber.cpp:1:
/opt/ros/humble/src/gtest_vendor/include/gtest/gtest.h: In instantiation of ‘testing::AssertionResult testing::internal::CmpHelperEQ(const char*, const char*, const T1&, const T2&) [with T1 = int; T2 = long unsigned int]’:
/opt/ros/humble/src/gtest_vendor/include/gtest/gtest.h:1554:23:   required from ‘static testing::AssertionResult testing::internal::EqHelper::Compare(const char*, const char*, const T1&, const T2&) [with T1 = int; T2 = long unsigned int; typename std::enable_if<((! std::is_integral<_Tp>::value) || (! std::is_pointer<_Dp>::value))>::type* <anonymous> = 0]’
/nav2_ros_gz/src/navigation2/nav2_costmap_2d/test/integration/test_costmap_subscriber.cpp:137:3:   required from here
/opt/ros/humble/src/gtest_vendor/include/gtest/gtest.h:1527:11: error: comparison of integer expressions of different signedness: ‘const int’ and ‘const long unsigned int’ [-Werror=sign-compare]
 1527 |   if (lhs == rhs) {
      |       ~~~~^~~~~~
cc1plus: all warnings being treated as errors
gmake[2]: *** [test/integration/CMakeFiles/test_costmap_subscriber_exec.dir/build.make:76: test/integration/CMakeFiles/test_costmap_subscriber_exec.dir/test_costmap_subscriber.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:1058: test/integration/CMakeFiles/test_costmap_subscriber_exec.dir/all] Error 2
gmake: *** [Makefile:146: all] Error 2
---
Failed   <<< nav2_costmap_2d [14.7s, exited with code 2]

Summary: 21 packages finished [21.2s]
  1 package failed: nav2_costmap_2d
  2 packages had stderr output: nav2_costmap_2d nav2_simple_commander
  22 packages not processed`

--- a/.devcontainer/Dockerfile
+++ b/.devcontainer/Dockerfile
@@ -61,7 +61,7 @@ USER $USERNAME
 RUN sudo usermod --append --groups video $USERNAME
 
 # Update all packages
-RUN sudo apt update & sudo apt upgrade -y
+RUN sudo apt update && sudo apt upgrade -y
 
 ## install python packages
 

--- a/README.md
+++ b/README.md
@@ -19,7 +19,7 @@ rosdep install -r --from-paths . --ignore-src --rosdistro $ROS_DISTRO -y
 ```
 colcon build --symlink-install --packages-skip nav2_system_tests
 ```
-## Source (already done in the dev container)
+## Source or open another terminal tab (as already sourced in the dev container bashrc)
 ```bash
 source /nav2_ros_gz/install/setup.bash
 ```

